catalyst-gateway/bin/src/service/api/registration/mod.rs

```diff
@@ -54,19 +54,21 @@ impl RegistrationApi {
         &self, pool: Data<&Arc<State>>,
         /// A Voters Public ED25519 Key (as registered in their most recent valid
         /// [CIP-15](https://cips.cardano.org/cips/cip15) or [CIP-36](https://cips.cardano.org/cips/cip36) registration).
-        #[oai(validator(max_length = 66, min_length = 66, pattern = "0x[0-9a-f]{64}"))]
+        #[oai(validator(max_length = 64, min_length = 64, pattern = "[0-9a-f]{64}"))]
```

Comment:
Is it needed to pass openapi linting ? Because it was added intentionally to have a following format `0x1b`

Reply:
The voting key example is 'a6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d1663'